textproc/most/Portfile

```diff
@@ -1,7 +1,7 @@
 PortSystem      1.0
 
 name            most
-version         5.1.0
+version         5.2.0
```

Comment:
5.2.0 is not mentioned on the project's web site but it's probably just an oversight. I've [asked the developer](https://github.com/jedsoft/most/issues/24) just to make sure.

Reply:
It's [here](https://www.jedsoft.org/releases/most/).

Reply:
Yes, I know it is. What I meant is that it is not mentioned [here](https://www.jedsoft.org/most/).